src/Database/PostgreSQL/PQTypes/Checks.hs

```diff
@@ -48,22 +49,37 @@ migrateDatabase options extensions domains tables migrations = do
   resultCheck =<< checkDBStructure (tableVersions : tables)
   resultCheck =<< checkTablesWereDropped migrations
   resultCheck =<< checkUnknownTables tables
+  resultCheck =<< checkExistanceOfVersionsForTables (tableVersions : tables)
 
   -- everything is OK, commit changes
   commit
 
 -- | Run checks on the database structure and whether the database
--- needs to be migrated.
+-- needs to be migrated. Will do a full check of DB structure
 checkDatabase
   :: forall m . (MonadDB m, MonadLog m, MonadThrow m)
   => [Domain] -> [Table] -> m ()
-checkDatabase domains tables = do
+checkDatabase = checkDatabase_ False
+
+-- | Same as `checkDatabase`, but will not failed if there are
```

Comment:
Ditto. Also backticks are not rendered as typewriter font in Haddock, please use `'foo'` or `@foo@` instead. See https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810770608.